feat(iam): added manage organization + quotas rework MTA-6194 (#6769)
scaleway/docs-content: pages/organizations-and-projects/how-to/manage-quotas.mdx

1. Click the **Quotas** tab of the [Organization Dashboard](https://console.scaleway.com/organization). The list of quotas filtered by product appears.
2. Click the arrow icon next to the product name to expand or collapse detailed quotas.
3. Click <Icon name="more" /> > **More info** next to the name of your resource of choice. A dialog box appears with the following quota information: Name, Technical name, Product, and Description.

Use the **Filters** section to search your quotas. Modifying one of the following options automatically updates the list of quotas:

1. Search by the precise name of the resource.
2. Specify the product you want to see the quotas for.
3. Specify the location of your resources. This includes regions, Availability Zones, or all locations.
4. Check the box to see used quotas only.
5. Click the **Clear all** button to reset the filters.

## Increase quotas

Quotas are assigned by default according to each account level.

There are two ways you can increase your account level:
    - By [validating your payment method](/billing/how-to/add-payment-method/#how-to-add-a-credit-card)
    - By [verifying your identity](/account/how-to/verify-identity/)

> For a detailed list of available quotas and the account levels necessary to increase quota capacity, as well as availability per location, see [Understanding Organization Quotas](/organizations-and-projects/additional-content/organization-quotas/).

Reviewer: ```suggestion
For a detailed list of available quotas and the account levels necessary to increase quota capacity, as well as availability per location, see [Scaleway account - Organization quotas](/organizations-and-projects/additional-content/organization-quotas/).
```

Reviewer: I'm suggesting this change because this seems to be the current title of the doc.

<Message type="note">

>   If you have already validated your payment method and identity and want to further increase your quota, you must [contact our Support team](https://console.scaleway.com/support/create).

Reviewer 2: On devrait dire Scaleway Support team ?

Author: Je vais même aligner avec les guidelines et mettre Scaleway Support. Merci :D

</Message>

## Add an Organization

> If you have been added to an existing Organization as a [Member](/iam/concepts/#member), you can add the organization to your list as follows:

Reviewer 2: ```suggestion
If an existing Organization adds you as a [Member](/iam/concepts/#member), you can add the organization to your list as follows:
```

> 1. In the header menu, click on the **Organization name** to see the list of Organizations.

Reviewer 2: ```suggestion
1. In the header menu, click the **Organization name** to display the list of Organizations.
```

> 2. Click the **Add Organization** button at the top of the list. You are redirected to the **Log in as an IAM Member** page.

Reviewer 2: ```suggestion
2. Click the **Add Organization** button at the top of the list. The **Log in as an IAM Member** page opens.
```

Reviewer 2: I wouldn't say "Click the xx button" but rather "Click xx"

> 3. From there, you can [Log in as an IAM Member](/iam/how-to/log-in-as-a-member/). Upon doing so, the corresponding Organization is added to your Organization list.

Reviewer 2: ```suggestion
3. [Log in as an IAM Member](/iam/how-to/log-in-as-a-member/). The Organization is added to your Organization list.
```

## Remove an Organization

> You can remove Organizations from the list at any time. This does not impact your permissions— logging back into a removed Organization makes it appear in the list again.

Reviewer: ```suggestion
You can remove Organizations from the list at any time. This does not impact your permissions — logging back into a removed Organization makes it appear in the list again.
```

> You can remove Organizations from the list. This does not impact your permissions; logging back into the removed Organization will make it appear in the list again.

Reviewer 2: ```suggestion
You can remove Organizations from the list at any time. This does not impact your permissions— logging back into a removed Organization makes it appear in the list again.
```

> 1. In the header menu, click on the **Organization name** to see the list of Organizations.

Reviewer 2: ```suggestion
1. In the header menu, click the **Organization name** to display the list of Organizations.
```

2. Click the **Manage Organization list** button at the bottom of the dropdown. The **Manage Organization list** dialog box appears.

> 3. Check one or several Organizations to remove. You cannot select the current Organization.

Reviewer 2: ```suggestion
3. Select one or several Organizations to remove. You cannot select the current Organization.
```

Reviewer 2: I would say "select" but I get why "check" so as you wish

Author: Select feels indeed more natural, maybe we should update the guidelines for checkboxes ^^

Reviewer 2: I thought we agreed upon using "select" in these cases

> 4. Click the **Remove from list** button to remove the selected Organizations. The page refreshes, and the removed Organizations are no longer displayed.

Reviewer 2: ```suggestion
4. Click the **Remove from list** button. The page refreshes, and the removed Organizations no longer display.
```
